### ISSUE TITLE
Tutorial: Getting Started – Includes | Include SLS, not ID

### ISSUE THREAD

**Issue by fportillo** [saltstack/salt#28595]
Hi, 

Issue observed in this version:

```
root@saltmaster:/srv/salt# salt --versions-report
                  Salt: 2015.5.3
                Python: 2.7.6 (default, Jun 22 2015, 17:58:13)
                Jinja2: 2.7.2
              M2Crypto: 0.21.1
        msgpack-python: 0.3.0
          msgpack-pure: Not Installed
              pycrypto: 2.6.1
               libnacl: Not Installed
                PyYAML: 3.10
                 ioflo: Not Installed
                 PyZMQ: 14.0.1
                  RAET: Not Installed
                   ZMQ: 4.0.4
                  Mako: Not Installed
               Tornado: 4.2.1
 Debian source package: 2015.5.3+ds-1trusty1
```
This doc https://docs.saltstack.com/en/getstarted/config/include.html states the following:

> Where ID and ID2 are the State IDs of the sections that you want to include. Note that includes use the State ID, not the state filename.

However, the behavior is exatly the opposite (lost some time to figure out because of the misleading information):

```
root@saltmaster:/srv/salt# pwd
/srv/salt
root@saltmaster:/srv/salt# ls
common  examples.sls  hello-world.sls  nettools.sls  top.sls
root@saltmaster:/srv/salt# cat hello-world.sls 
say_hello_world:
  cmd.run:
    - name: "echo 'Hello World'"
root@saltmaster:/srv/salt# cat examples.sls 
include:
  - hello-world

Pillars hello world:
  cmd.run:
    - name: echo {{pillar['foo']}}
root@saltmaster:/srv/salt# salt '*' state.apply examples
minion2:
    ----------
    cmd_|-Pillars hello world_|-echo bar_|-run:
        ----------
        __run_num__:
            1
        changes:
            ----------
            pid:
                12003
            retcode:
                0
            stderr:
            stdout:
                bar
        comment:
            Command "echo bar" run
        duration:
            2.39
        name:
            echo bar
        result:
            True
        start_time:
            22:42:58.520315
    cmd_|-say_hello_world_|-echo 'Hello World'_|-run:
        ----------
        __run_num__:
            0
        changes:
            ----------
            pid:
                12002
            retcode:
                0
            stderr:
            stdout:
                Hello World
        comment:
            Command "echo 'Hello World'" run
        duration:
            2.505
        name:
            echo 'Hello World'
        result:
            True
        start_time:
            22:42:58.517719
minion1:
    ----------
    cmd_|-Pillars hello world_|-echo foo_|-run:
        ----------
        __run_num__:
            1
        changes:
            ----------
            pid:
                11914
            retcode:
                0
            stderr:
            stdout:
                foo
        comment:
            Command "echo foo" run
        duration:
            2.925
        name:
            echo foo
        result:
            True
        start_time:
            22:42:59.683230
    cmd_|-say_hello_world_|-echo 'Hello World'_|-run:
        ----------
        __run_num__:
            0
        changes:
            ----------
            pid:
                11913
            retcode:
                0
            stderr:
            stdout:
                Hello World
        comment:
            Command "echo 'Hello World'" run
        duration:
            2.65
        name:
            echo 'Hello World'
        result:
            True
        start_time:
            22:42:59.680488
root@saltmaster:/srv/salt# vim examples.sls 
root@saltmaster:/srv/salt# cat examples.sls 
include:
  - say_hello_world

Pillars hello world:
  cmd.run:
    - name: echo {{pillar['foo']}}
root@saltmaster:/srv/salt# salt '*' state.apply examples
minion1:
    - Specified SLS say_hello_world in saltenv base is not available on the salt master or through a configured fileserver
minion2:
    - Specified SLS say_hello_world in saltenv base is not available on the salt master or through a configured fileserver
```

TLDR: include using state filename without .sls works, but including state ID does not.

Please fix tutorial docs. It will help a lot people new to salt desiring to get started with it. :)

**Comment by ssgward** [comment]
Thanks for the report. We'll get this assigned out.

**Comment by Granjow** [comment]
@fportillo Maybe you can rename this bug to something like
`Tutorial: Getting Started – Includes | Include SLS, not ID`
so that it is a bit easier to find.

Thanks for reporting, I just had the same question.

**Comment by fportillo** [comment]
@Granjow : thanks for the advice, done. :)

**Comment by jacobhammons** [comment]
@fportillo I've made this update, thanks for submitting!

**Comment by Granjow** [comment]
Thanks for the update!
